site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -1167,6 +1167,11 @@ public function ajaxGetGradedGradeable($gradeable_id, $anon_id = '', $all_peers
         if ($submitter_id === false) {
             return;
         }
+        
+        $section = $this->core->getQueries()->getUserById($submitter_id)->getRotatingSection();
+        if ($gradeable->isGradeByRegistration()) {
+            $section = $this->core->getQueries()->getUserById($submitter_id)->getRegistrationSection();
+        }
```

Comment:
Use an if/else here, so that we're not wasting a potential query.

Reply:
Added the if else